### DISCUSSION TITLE
How to force image from parsed markdown to use Image form next/image ?

### DISCUSSION THREAD

**Discussion by iyansr** [vercel/next.js#18500] [Help]
I just want to ask, how to force image from parsed markdown to use Image form next/image ?

I have markdown blog that contain image, and i have no idea how to change default `<img>` tag that parsed by `marked` into `<Image>` tag from next.js

thanks

**Comment by jurajkrivda** [comment]
```
<ReactMarkdown
      {...props}
      renderers={{
        img: Image,
      }}
    />
```

Where Image is next/image
